src/Magento/FunctionalTestingFramework/Console/GenerateTestsCommand.php

```diff
@@ -78,9 +80,8 @@ protected function execute(InputInterface $input, OutputInterface $output)
         $json = $input->getOption('tests');
         $force = $input->getOption('force');
         $time = $input->getOption('time') * 60 * 1000; // convert from minutes to milliseconds
-        $debug = $input->getOption('debug');
+        $mode = $input->getOption('debug');
```

Comment:
I'm not 100% sure how you can currently go from `--debug` to `self::MODE_DEVELOPER`, I may be missing something. Unless the intention is for me to input `--debug developer` or something, in which case we shoudl change the desc to show the required options.

If not, we can map true/false all existing --debug cases (we'd need to keep it string input though).

`--debug true` == `debug = true` == `self::MODE_DEVELOPER`
`--debug false` == `debug = false ` == `self::MODE_PRODUCTION`
`no debug flag` == `debug = null` == `self:MODE_DEFAULT`

Reply:
Updated as per recent discussions
- in GenerateTestsCommand `$debug` input option is set to optional; 
`--debug` == LEVEL_DEVELOPER (backward compatibility)
`--debug developer` == LEVEL_DEVELOPER
`--debug none` == DEBUG_NONE
`no debug flag` == LEVEL_DEFAULT